shared/elements/gaia_lowstorage_dialog/script.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+/* global ComponentUtils, MozActivity */
+
+/**
+ * The gaia-confirm component displays a dialog in which the user has a
+ * choice to confirm or cancel the action. It may be displayed along with a
+ * title, description, and image. Buttons may also be configured.
+ * @requires GaiaButtons
+ */
+window.GaiaLowstorageDialog = (function(win) {
+  // Extend from the HTMLElement prototype
+  var proto = Object.create(HTMLElement.prototype);
```

Comment:
I'm new to WebComponents, but can't we use ```gaiaComponent.register``` here, like other web components do to hide some of the complexity [1]?

[1] https://github.com/gaia-components/gaia-checkbox/blob/cf04bb03f1b8390a6d973bdc2e37edcf246a5d61/gaia-checkbox.js#L15